icarus_backend/user/UserViews.py

```diff
@@ -128,4 +125,56 @@ def activate(request, uidb64, token):
         # return redirect('home')
         return HttpResponse('Thank you for your email confirmation. Now you can login your account.')
     else:
-        return HttpResponse('Activation link is invalid!')
\ No newline at end of file
+        return HttpResponse('Activation link is invalid!')
+
+
+@api_view(['GET'])
+def forgot_password(request):
+    email = request.query_params.get('email')
+    user = User.objects.filter(email=email).first()
+    if not user:
+        response_data = {'message': 'No user with this email exists.'}
```

Comment:
hellllll no! This is how they mine your users. This is solid no, you should always say email will be sent if the user exists. 

Lets say that I have a list of hacked emails, i write a python script and run them through your forgot password. Many of them will give no user with this email exists, and the ones that are registered will say password reset email is being sent. Assuming people use the same emails, they might gain access to the users ICARUS profile